src/sentry/testutils/hybrid_cloud.py

```diff
@@ -93,3 +96,41 @@ def cb(service: Any, method_name: str, *args: Sequence[Any], **kwds: Mapping[str
         yield
     finally:
         hc_test_stub.cb = None
+
+
+class HybridCloudTestMixin:
+    @exempt_from_silo_limits()
+    def assert_org_member_mapping(self, org_member: OrganizationMember, expected=None):
```

Comment:
These helpers should probably test that both the organization member and the org member mapping exists. We're testing state synchronization after all